### DISCUSSION TITLE
List of favs/"liked" apps?

### DISCUSSION THREAD

**Discussion by nicoleritzer** [NeoApplications/Neo-Store#267] [Ideas]
Since the Neo Store provides the ability to heart/like/fav apps (to the left of the "install" button, usually in a dropdown menu with the "share" button), is there a way to see a list of my favs/likes? 

If this doesn't exist, will it be implemented? I'd like some way to keep a list of apps (without necessarily downloading them), especially since F-Droid and its forks don't work with AppWatcher. Thanks!

**Comment by machiav3lli** [comment]
There's already a favorites filter in the current rc, a release will come soon enough.

In the future there'll be also options to export/import favorites among other actions…

**Comment by nicoleritzer** [comment]
Thanks for the info!

**Comment by machiav3lli** [comment]
The feature to export/import favorites has been added in the latest 0.9.13
